addons/binding/org.openhab.binding.heos/ESH-INF/thing/HeosGroup.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="group">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+		<label>HEOS Group</label>
+		<description>A group of HEOS Player</description>
+
+		<channels>
+			<channel id="Control" typeId="control" />
+			<channel id="Volume" typeId="volume" />
+			<channel id="Mute" typeId="mute" />
+			<channel id="Inputs" typeId="input" />
+			<channel id="Title" typeId="title" />
+			<channel id="Interpret" typeId="interpret" />
+			<channel id="Album" typeId="album" />
+			<channel id="Cover" typeId="cover" />
+			<channel id="CurrentPosition" typeId="currentPosition" />
+			<channel id="Duration" typeId="duration" />
+			<channel id="Type" typeId="type" />
+			<channel id="Station" typeId="station" />
+			<channel id="PlayUrl" typeId="playUrl" />
+			<channel id="Ungroup" typeId="ungroup" />
+			<channel id="Shuffel" typeId="shuffleMode" />
+			<channel id="RepeatMode" typeId="repeatMode" />
+		</channels>
+
+		<config-description>
+			<parameter name="gid" type="text" readOnly="true">
+				<label>Group ID</label>
+				<description>The internal group ID</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="name" type="text" readOnly="true">
+				<label>Group Name</label>
+				<description>The name of the group</description>
+				<required>false</required>
+			</parameter>
+			<parameter name="leader" type="text" readOnly="true">
+				<label>Leading HEOS PID</label>
+				<description>The PID of the group leading HEOS player</description>
+				<required>false</required>
+			</parameter>
+			<parameter name="type" type="text" readOnly="true">
+				<label>Group or Player</label>
```

Comment:
Please specify all possible values so that the user is presented a dropdown

Reply:
If I start thinking of it, there shouldn't be an option. For groups only "Group" is allowed. For players only "Player" should be allowed.

Reply:
But where is the parameter then needed for?

Reply:
It was needed for the binding to know either if it is a player or a group. But I will use the handler class of the thing instead to identify the thing type and remove the parameters from the group and player.xml